Source/Scene/Primitive.js

```diff
@@ -994,15 +994,11 @@ define([
     function depthClampVS(vertexShaderSource) {
         var modifiedVS = ShaderSource.replaceMain(vertexShaderSource, 'czm_non_depth_clamp_main');
         modifiedVS +=
-            '#ifdef GL_EXT_frag_depth\n' +
```

Comment:
As an explicit optimization, we usually put the `#ifdef` everywhere and do not rely, for example, on the GLSL compiler to optimize out unused code.

Perhaps add a comment here that this is used for frag depth, but it is not in an `#ifdef` to workaround an Edge bug.

Also submit an Edge bug please.